BigCommerce/Integration/04_ManageProductData.md

```diff
@@ -0,0 +1,99 @@
+The following procedure describes how to check the images in the *PIM* module. If you only have the *Omni-Channel* module in use, you can check the image accordingly via *Offers > Select offer > Edit offer*.
+
+
+#### Prerequisites
+
+You have assigned more than one image per product variant.
+
+#### Procedure
+
+*PIM > Products > Tab LIST*
+
+![Product images](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/PIMCheckImages.png "[Product images]")
+
+1. Click the product variant you want to check and click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button.   
+  The *Edit product* view is displayed. The *Attributes* tab is preselected.
+
+    ![Edit product](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditProduct.png "[Edit product]")
+
+2. Click the *Files and Images* entry of the *Basic Data* attribute group or the customer-specific attribute group in which you have defined the images.   
+   The *Files and images* section is displayed. 
+
+   ![Files and images](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/PIMImageAndFiles.png "[Files and images]")
+
+3. Verify that the image at the first position is suitable for your *BigCommerce* store. If not, drag and drop the desired image to the first position.
+
+4. Verify that the *Alt text* field is filled for the image on first position. If you must change the existing entry, you have the following options: 
+    - You can change the entry. This might have impact on other connections you have in use.
+    - If and want to keep the product data in this scope unchanged, copy your products to a specific *BigCommerce* sales channel. <!---Stimmt das, sollte man das so tun?-->
+
+5. If applicable, click the [SAVE] button to save your changes.
+
+
+
+## Configure color&dash;swatch variant option
+
+Configure the color-swatch variant option for the *BigCommerce* products for which the options provided in the connection settings are not suitable. 
+A color-swatch option in *BigCommerce* is an option with which the store customers can select a color combination in the *BigCommerce* storefront. For example, you want to offer T-shirts that are available with the color combinations black/white, white/green, and black/green. *BigCommerce* supports up to three colors per color swatch value.   
+ For detailed information on the standard selection options, see [Configure BigCommerce connection](../Integration/01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
+
+#### Prerequisites
+
+- To add a variant option as a color-swatch, the defining attribute in the variant set needs to be of the *Tree node* data type.
```

Comment:
Vorschlag: 
- The defining attribute in the variant set is of the *Tree node* type.

Prerequisites sind Sachen, die man schon gemacht hat / machen muss, bevor man mit der Procedure anfängt, daher würde ich es anders formulieren. Oder kann man das im Nachhinein ändern/einstellen? Dann vielleicht unter dem/im jeweiligen Schritt?